templates/uwsgi_systemd.service.erb

```diff
@@ -0,0 +1,15 @@
+#
+# This file is managed by puppet class 'uwsgi'
+#
+# uWSGI Emperor process systemd service
+#
+[Unit]
+Description=uWSGI Emperor
+After=network.target
+
+[Service]
+Type=simple
+ExecStart=/usr/sbin/uwsgi --die-on-term --ini <%= @config_file %>
```

Comment:
I'd just go with something like "binary_location" and include the `uwsgi` component. Thoughts?